cc3d/core/PySteppables.py

```diff
@@ -755,7 +755,10 @@ def steering_param_dirty(self, name=None):
         with pg.steering_panel_synchronizer:
 
             if name is not None:
-                return self.get_steering_param(name=name).dirty_flag
+                try:
```

Comment:
maybe we can introduce set_dirty_flag(param_name) function and move the code there?